src/components/List/list.js

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import "./list.css";
+import Input from "../Input/input";
+
+const List = () => {
+  const [indexCheck, setIndexCheck] = useState([]);
+  const [list, setList] = useState(["xyz", "abc"]);
+  const [printInput, setPrintInput] = useState("");
+
+  let handleClick = (event, index) => {
+    const { checked } = event.target;
+    if (checked === true) {
```

Comment:
if (checked === true) { ??? is === necessary @PuniaMohit ?

Reply:
sir, it is neccessay because i am making array which stores index of selected task. If task is selected than checked is true and array gets index else checked is false index is removed from array.